plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -174,10 +174,16 @@
      * The OpenShift deploy timeout in seconds:
      * See this issue for background of why for end users on slow wifi on their laptops
      * DeploymentConfigs usually barf: https://github.com/openshift/origin/issues/10531
+     *
+     * Lets default to 2 hours which should be plenty of time on a slow connection to download
+     * all the images for a YAML/JSON manifest (which can contain many separate images)
      */
-    @Parameter(property = "fabric8.openshift.deployTimeoutSeconds")
+    @Parameter(property = "fabric8.openshift.deployTimeoutSeconds", defaultValue = "7200")
```

Comment:
Agreed. Do you know when `Deployments` will be supported on OpenShift ? (BTW wonder also how we should deal with backwards compatibility with people using older OpenShift installations).

I suggest to continue from here on with:

* increasing the timeout (in a separate PR)
* me checking how to avoid the image trigger for non-s2i scenarios and how to refactor the resource generation as described in #678